FirstSolution/Part17Task1P2/Triangle.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace Part17Task1P2
+{
+    public class Triangle : ISimpleNAngle
+    {
+        private const int numberOfSides = 3;
+
+        public Triangle(double angleBetweenFloorAndAdjacentSide, double[] lengthOfSides)
+        {
+            if (
+                angleBetweenFloorAndAdjacentSide > 0 &&
+                angleBetweenFloorAndAdjacentSide < 180 &&
+                lengthOfSides[0] > 0 &&
+                lengthOfSides[1] > 0 &&
+                lengthOfSides[2] > 0 &&
+                lengthOfSides[0] + lengthOfSides[1] > lengthOfSides[2] &&
+                lengthOfSides[1] + lengthOfSides[2] > lengthOfSides[0] &&
+                lengthOfSides[2] + lengthOfSides[0] > lengthOfSides[1]
+                )
+            {
+                Height = lengthOfSides[1] * Math.Sin(Math.PI * angleBetweenFloorAndAdjacentSide / 180);
+                Floor = lengthOfSides[0];
+                NumberOfSides = numberOfSides;
+                LengthOfSides = lengthOfSides;
+                Area = (Height * Floor) / 2;
+                foreach (var i in lengthOfSides)
```

Comment:
Rename variable to "lengthOfSide". Don't use abstract names in foreach

Reply:
Done